storage/remote/generic/generic.proto

```diff
@@ -15,9 +15,8 @@ syntax = "proto3";
 
 package generic;
 
-
 message Sample {
-  double value = 1;
```

Comment:
I did that to stay consistent with our existing `metrics.proto` formatting and the discussion around that on https://github.com/prometheus/client_model/pull/14/files. Don't really care about it much either way, but then we should change it everywhere.